llvm/tools/llvm-cov/SourceCoverageViewHTML.cpp

```diff
@@ -88,13 +88,123 @@ const char *BeginHeader =
     "<meta name='viewport' content='width=device-width,initial-scale=1'>"
     "<meta charset='UTF-8'>";
 
+const char *JSForCoverage =
+    R"javascript(
+
+function next_uncovered(selector, reverse, scroll_selector) {
```

Comment:
Could we unify the cursor? I guess users want to jump to the prev/next *visible* issue.
For example, "Jump to the prev/next another uncovered line in sight when the cursor is navigated by branch."

Reply:
I don't know how, at least not without really big refactor in how the HTML is generated.